Sources/Apodini/Properties/Parameter.swift

```diff
@@ -70,8 +76,10 @@ public struct Parameter<Element: Codable>: Property {
     ///   - options: Options passed on to different interface exporters to clarify the functionality of this `@Parameter` for different API types
     public init(wrappedValue defaultValue: Element, _ name: String? = nil, _ options: Option...) {
         self.defaultValue = defaultValue
-        self.name = name
         self.options = PropertyOptionSet(options)
+        defer { // swiftlint:disable:this inert_defer
+            self.name = name
+        }
```

Comment:
I am not sure why you use a defer statement here. As this would be the last operation it will be executed last anyways the defer would not be needed here.
```suggestion
        self.name = name
```

Reply:
Same as above. 👍